plone/app/caching/profiles/default/controlpanel.xml

```diff
@@ -10,7 +10,7 @@
         appId="plone.app.caching"
         category="plone-advanced"
         condition_expr=""
-        icon_expr="string:$portal_url/++resource++plone.app.caching.gif"
+        icon_expr="string:skip-forward"
```

Comment:
how about using "speedometer" or "speedometer2"? but don't have a strong opinion ;-)

Reply:
I would use the same icon type but neither I've a strong opinion ;-)

Reply:
`+resource++plone.app.caching.gif` is from plone 4

In Plone 5 we used the webfont with a `gage`icon  ;-)